CHANGELOG.md

```diff
@@ -3,6 +3,11 @@ Changelog
 
 Unreleased
 ----------
+### Added
+* Include a postcss plugin to apply a scoped theme as the default theme and to remove unlisted supported themes.
+
+### Changed
+* For testing, only aggregate the default theme.
```

Comment:
I'm noticing we are leasing our test themes 😬 can we add to the npm ignore?

Reply:
you mean we're releasing the generated themes folder right? the tests folder is already ignored.